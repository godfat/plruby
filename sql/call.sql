CREATE OR REPLACE FUNCTION plruby_call()
RETURNS text AS $$
  p('CALL')
$$ LANGUAGE plruby IMMUTABLE STRICT;
SELECT plruby_call();
