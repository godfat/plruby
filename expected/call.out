CREATE OR REPLACE FUNCTION plruby_call()
RETURNS text AS $$
  p('CALL')
$$ LANGUAGE plruby IMMUTABLE STRICT;
SELECT plruby_call();
 plruby_call 
-------------
 CALL
(1 row)

